data-prepper-plugins/mutate-string-processors/src/main/java/com/amazon/dataprepper/plugins/processor/mutatestring/SplitStringProcessor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.mutatestring;
+
+import com.amazon.dataprepper.metrics.PluginMetrics;
+import com.amazon.dataprepper.model.annotations.DataPrepperPlugin;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.processor.Processor;
+
+import java.util.List;
+
+@DataPrepperPlugin(name = "split_string", pluginType = Processor.class, pluginConfigurationType = SplitStringProcessorConfig.class)
+public class SplitStringProcessor extends AbstractStringProcessor<SplitStringProcessorConfig.Entry> {
+
+    private final List<SplitStringProcessorConfig.Entry> entries;
+
+    public SplitStringProcessor(PluginMetrics pluginMetrics, StringProcessorConfig<SplitStringProcessorConfig.Entry> config) {
```

Comment:
The parameters in all of these functions can be `final`

Reply:
Yes, thanks for catching this